src/postgres/9-updates.sql

```diff
@@ -314,4 +314,14 @@ SET edprole_id = '804b335c-f191-4d26-9b98-1ec1cb62b97d'
 WHERE edprole_id = '29ccab4b-65e2-4764-83ec-77375d29af39';
 
 delete from edprole_privilege where edprole_id='29ccab4b-65e2-4764-83ec-77375d29af39';
-delete from edprole where id='29ccab4b-65e2-4764-83ec-77375d29af39'
\ No newline at end of file
+delete from edprole where id='29ccab4b-65e2-4764-83ec-77375d29af39'
+
+--6/10/2024 Move management review step after UWG review step in GES DISC community workflow
+
+INSERT INTO step(step_id, step_name, type, data) VALUES ('c81066db-0566-428d-87e8-94169ce5a9b9', 'data_publication_request_form_uwg_review', 'review', '{"rollback":"data_publication_request_form_review","type": "review","form_id":"19025579-99ca-4344-8610-704dae626343"}');
```

Comment:
Because these steps already exist, these should be update commands. The 9-updates.sql file is used to describe what changes need to be deployed to our SIT/UAT/PROD environments while the 7-seed.sql file is used to describe what should be deployed on initialization of a new database (usually only local).

Reply:
Ohh I see...Thanks for adding Update commands. I appreciate it!